spec/services/api/v1.2/order_process_dissociator_spec.rb

```diff
@@ -0,0 +1,46 @@
+describe Api::V1x2::Catalog::OrderProcessDissociator do
+  describe "#process" do
+    let(:order_process) do
+      create(:order_process,
+             :before_portfolio_item => before_portfolio_item,
+             :after_portfolio_item  => after_portfolio_item)
+    end
+    let(:before_portfolio_item) { create(:portfolio_item) }
+    let(:after_portfolio_item) { create(:portfolio_item) }
```

Comment:
@eclarizio Should we have case where the before and after point to the same portfolio item. Since the user is going to be picking the item from a drop down they could pick both the before and after to be the same.

Reply:
@mkanoor Yeah, that's true, although I don't think there needs to be a case for it, it's not handled any differently and it would still work as intended, feels like that would just be testing that rails handles their associations properly, which I think we trust 😄